README.md

```diff
@@ -1,2 +1,154 @@
+   ```
+   cd school_library
+   ```
+
+### Prerequisites <a name="prerequisites"></a>
+- An integrated development environment (IDE) like Visual Studio Code.
+- Ruby installed on your machine.
+- Git installed on your machine.
+- A GitHub account for version control.
+
+# Usage of Ruby <a name="usage"></a>
+
+Follow these steps to use the program:
+
+1. Open your terminal.
+2. Navigate to the directory where the program files are located using the `cd` command.
+
+```bash
+cd /path/to/your/program/files
+```
+
+3. Start the Interactive Ruby (IRB) console by typing `irb` and pressing Enter.
+
+```bash
+irb
+```
+
+4. Inside the IRB console, load the `test.rb` file to execute the program.
+
+```ruby
+load 'test.rb'
+```
+
+5. The program will run, and you will see the output in the console.
+
+```
+maximilianus
+Maximilianus
+Maximilian
+Classroom label: Math
+Student 1: Alice (16 years old)
+Student 2: Bob (17 years old)
+Rental date: 2023-08-30
+Rental person: John
+Rental book: The Great Gatsby
+Person's rentals: ["The Great Gatsby"]
+Book's rentals: ["John"]              
+=> true
+```
+
+That's it! You've successfully executed the program using IRB.
+
+Remember to replace `/path/to/your/program/files` with the actual path to the directory where your program files are located.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors <a name="authors"></a>
+👤 **Nicolas Bolañez**
+- GitHub: [@Nico](https://github.com/Zven9)
+- LinkedIn: [@Nico](https://www.linkedin.com/in/nico-emiliano/)
+
+
+## ❓ FAQ <a name="faq"></a>
+
+**Q: How do I add a new student or teacher to the library?**
+A: To add a new student or teacher to the library, you can leverage the methods provided in the Ruby script. There will be functions designed to facilitate the addition of new individuals to the library's database. Make sure to follow the usage instructions in the appropriate sections of the README.
+
+**Q: Is there a plan to add more features to this application?**
+A: Yes, the development team has plans to enhance this application's capabilities. One upcoming feature is to make the application executable via the command line, providing more convenience and accessibility to users.
+
+**Q: How can I contribute to the School Library (OOP) project?**
+A: Contributions are highly encouraged! If you have ideas, suggestions, or identify issues, please feel free to submit them through the project's GitHub repository. Refer to the contributing guidelines for more details.
+
+**Q: Where can I find more information about the Ruby programming language?**
+A: For more information about Ruby, its features, and its community, visit the official [Ruby website](https://www.ruby-lang.org/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome! Feel free to check the [issues page](https://github.com/Zven94/OOP-school-library/).
```

Comment:
- [ ] I also suggest you please use the correct link to you issue page as it makes you README more professional. `https://github.com/Zven94/OOP-school-library/issues`